src/mute.js

```diff
@@ -31,13 +30,16 @@ export default class Mute extends React.Component<MuteProps> {
         this.filter.removeWords(removeWord);  
       });
     }
-    
-
   }
 
   render() {
-    const { text, children } = this.props;
+    const { children } = this.props;
     
-    return this.props.children(this.filter.clean(text));
+    return (
+      // $FlowFixMe
```

Comment:
What's Flow complaining about here?

Reply:
```
/Users/amathur/workspace/react-mute/src/mute.js:39
  37 |
  38 |     return (
> 39 |       <React.Fragment>
     |              ^^^^^^^^ Cannot get `React.Fragment` because property `Fragment` is missing in object type [1].
  40 |         {this.filter.clean(children)}
  41 |       </React.Fragment>
  42 |     );
```

Reply:
Probaby because you need to `import { Fragment } from 'react';`.

Reply:
it was that 🙂 Thanks